operator/pkg/util/patcher/pather.go

```diff
@@ -109,6 +119,9 @@ func (p *Patcher) ForStatefulSet(sts *appsv1.StatefulSet) {
 		patchVolumeForStatefulSet(sts, p.volume)
 	}
 
+	if p.resources.Size() > 0 {
+		sts.Spec.Template.Spec.Containers[0].Resources = p.resources
```

Comment:
ditto, as we need to explain why put the resource requirements to the first container.

Reply:
done